spec/workers/users/resave_articles_worker_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe Users::ResaveArticlesWorker, type: :worker do
+  include_examples "#enqueues_on_correct_queue", "medium_priority", 1
+
+  describe "#perform" do
+    let(:worker) { subject }
+
+    context "with comment" do
```

Comment:
I think you mean "with user" here and "without user" below 😉

Been there, done that before! hehe

Reply:
😊